README.md

```diff
@@ -101,7 +101,7 @@ offending character is indicated by the number `8` after the colon. Change the
 variable `value_nodes` to `valueNodes` in your source file to eliminate this
 error.
 
-To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](https://github.com/w3c/aria-practices/blob/master/.eslint.json) file in the root of the project.
+To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](https://github.com/w3c/aria-practices/blob/main/.eslint.json) file in the root of the project.
```

Comment:
Not sure if the relative link works with dotfiles in the Markdown auto-linking or not
```suggestion
To see the complete list of style rules that are applied by ESLint, review the [.eslint.json](.eslint.json) file in the root of the project.
```

Reply:
> Could use the relative link, but I'm not sure if the relative link works with dotfiles in the Markdown auto-linking or not

I didn't see anything in the documentation that indicates that they wouldn't. https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes